app/src/main/java/com/lagradost/cloudstream3/ui/player/DownloadFileGenerator.kt

```diff
@@ -59,7 +62,19 @@ class DownloadFileGenerator(
         offset: Int
     ): Boolean {
         val meta = episodes[currentIndex + offset]
-        callback(null to meta)
+
```

Comment:
Add `If (meta.uri == Uri.EMPTY)`, as we can skip this step if uri is supplied.

Reply:
Done